src/test/java/StringManipulationTest.java

```diff
@@ -86,26 +102,50 @@ public void testGeSubStrings1() {
         assertEquals(sStings[0], "my");
         assertEquals(sStings[1], "string");
     }
-
+    //Checks for index out of bound exception being thrown
     @Test
     public void testGeSubStrings2() {
-        fail("Not yet implemented");
+    	manipulatedstring.setString("This is my string");
+    	Assertions.assertThrows(IndexOutOfBoundsException.class, () -> {manipulatedstring.getSubStrings(1, 5);});
     }
+    //checks for startWord being <= 0 throwing an exception
     @Test
     public void testGeSubStrings3() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("test with many words for exceptions");
+        Exception exception = assertThrows(IllegalArgumentException.class, () -> {manipulatedstring.getSubStrings(0, 1);});
+        String expected = "startWord must be greater than zero";
+        String actual = exception.getMessage();
+        assertTrue(actual == expected);
```

Comment:
Other test cases use **assertEquals** to check the actual and expected value, but this one uses **assertTrue**, in this way, we cannot see the difference between the actual and expected value.